ui/index.html

```diff
@@ -57,6 +57,7 @@
 
 		<link rel="manifest" href="/manifest.json" />
 		<meta name="theme-color" content="#141414" />
+		<title></title>
```

Comment:
@panagiotisbellias have you tested this code?

Reply:
@lifeparticle No, please confirm that I can run the application navigating to the ui directory and running `npm install`. Also, please provide the proper NodeJS version so I can test the code

Thanks,
Panagiotis

Reply:
@panagiotisbellias please follow the steps here, https://github.com/lifeparticle/binarytree/blob/main/ui/CONTRIBUTING.md

Thanks

Reply:
@lifeparticle I'm on it, thanks

Reply:
@lifeparticle I'm facing issues installing yarn. The v3.6.1 cannot be found either from Windows or WSL (Ubuntu 22.04)

Reply:
@panagiotisbellias can you please run the following command in your Terminal:

```shell
yarn --version
```

Did you follow this [guide](https://yarnpkg.com/getting-started/install) to install yarn 3?

Reply:
Sorry, I missed this.. Now I'm ok, moving on

Reply:
Now my issue is this:
`PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn set version 3.6.1
➤ YN0000: Retrieving https://repo.yarnpkg.com/3.6.1/packages/yarnpkg-cli/bin/yarn.js
➤ YN0000: Saving the new release in .yarn/releases/yarn-3.6.1.cjs
➤ YN0000: Done in 0s 793ms
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn --version
3.6.1
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn dev
Internal Error: root-workspace-0b6124@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
    at V0.getCandidates (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:435:5145)
    at kf.getCandidates (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:391:1264)
    at C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8033
    at df (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:11070)
    at ge (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8013)
    at async Promise.allSettled (index 0)
    at async io (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:10398)
    at async C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8545
    at async ti.startProgressPromise (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:47579)
    at async je.resolveEverything (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:6573)
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn install --immutable
➤ YN0070: Migrating from Yarn 1; automatically enabling the compatibility node-modules linker �

➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed

➤ YN0000: ┌ Post-resolution validation
➤ YN0000: │ @@ -1,4 +1,11 @@
➤ YN0028: │ -# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
➤ YN0028: │ -# yarn lockfile v1
➤ YN0028: │ +# This file is generated by running "yarn install" inside your project.
➤ YN0028: │ +# Manual changes might be lost - proceed with caution!
➤ YN0000: │
➤ YN0028: │ +__metadata:
➤ YN0028: │ +  version: 6
➤ YN0000: │
➤ YN0028: │ +"root-workspace-0b6124@workspace:.":
➤ YN0028: │ +  version: 0.0.0-use.local
➤ YN0028: │ +  resolution: "root-workspace-0b6124@workspace:."
➤ YN0028: │ +  languageName: unknown
➤ YN0028: │ +  linkType: soft
➤ YN0000: │
➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
➤ YN0000: └ Completed
➤ YN0000: Failed with errors in 0s 22ms
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn dev
Internal Error: root-workspace-0b6124@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
    at V0.getCandidates (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:435:5145)
    at kf.getCandidates (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:391:1264)
    at C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8033
    at df (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:11070)
    at ge (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8013)
    at async Promise.allSettled (index 0)
    at async io (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:10398)
    at async C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:8545
    at async ti.startProgressPromise (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:390:47579)
    at async je.resolveEverything (C:\Users\pbell\AppData\Local\node\corepack\yarn\3.6.1\yarn.js:439:6573)
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn install
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed
➤ YN0000: Done in 0s 49ms
PS C:\GitRepositories\personal\open-source-contribution\osc-binarytree> yarn dev
Usage Error: Couldn't find a script named "dev".

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...`

Reply:
@panagiotisbellias make sure that you're inside the `ui` folder

Reply:
Ok, now I did it with some warnings though. I see that the title is gone this way so i'm adding the correct title in the html element. This is good to be configurable though reading the title from the previous location

Reply:
Hi, if more actions are needed by me please inform me or close this PR. Thanks

Reply:
@panagiotisbellias, please pull the latest from the main and review your changes. For example, 

- Does this file need any cleanup? `ui/src/components/Layouts/PageGrid/PageGrid.tsx`
- Does the title change for every page? `ui/index.html`

Thanks

Reply:
Latest changes generated merge confilcts with my side branch, so I'm closing this pull request and open a new one for the same issue when time

Thanks